services/QuillLMS/client/app/bundles/Staff/styles/rule_analysis.scss

```diff
@@ -12,6 +12,34 @@
     }
   }
 
+  .date-selection-container {
+    display: flex;
+    align-items: baseline;
+    margin-top: 16px;
+    .date-picker-label {
+      margin-right: 8px;
+    }
+    .start-date-picker {
+      margin-right: 22px;
+    }
+    .end-date-picker {
+      margin-right: 16px;
+    }
+    .react-datetime-picker__inputGroup {
+      padding-top: 2px;
+    }
+    .react-datetime-picker__inputGroup__input {
+      height: auto;
+    }
+  }
```

Comment:
I think there is some repeated code between this and the other places you're updating the datepicker CSS - it would be nice to put that in a shared file.

Reply:
good point, I'll combine them into a shared file